nodejs/week3/homework-template/src/backend/api/meals.js

```diff
@@ -0,0 +1,85 @@
+const express = require("express");
+require("express-async-errors");
+const router = express.Router();
+const knex = require("../database");
+const { sum, limit } = require("../database");
+
+//to inser a new meal
+router.post("/", async (req, res) => {
+	const newMeal = {
+		title: req.query.title,
+		description: req.query.description,
+		location: req.query.location,
+		for_when: req.query.for_when,
+		max_reservations: req.query.max_reservations,
+		price: req.query.price,
+		created_date: req.query.created_date,
+	};
+	await knex("meals").insert(newMeal);
+	res.send(`new meal inserted: ${newMeal.title}`);
```

Comment:
A common pattern in REST when you create a new resource or entity like a meal is to return the object itself including its new id. That way the client that is using this new meal can use that id in the future if it needs to delete it, update it or obtain updates done by other clients.